config.toml

```diff
@@ -1,26 +1,40 @@
 URL = "http://eplus.jp/event_name"
-Browser = "Chrome"
 
 [[Ballots]]
 Credentials = { username = "username@email.com", password = "password" }
 
 Codes = ["aaa", "bbb"]
-Days = "Both"
+# You can specify a list or you can just write "All" if you want to apply for all sessions.
+# Examples:
+# Sessions = ["昼公演"]
+# Sessions = ["昼公演", "夜公演"]
+# Sessions = "All"
+Sessions = "All"
 Pair = true
+# If your renban list is A, B, C, ..., the value you should pick is:
+# A -> 1
+# B -> 2
+# C -> 3
+# ...
+# You can also find this as the "No" of the person on
+# https://member.eplus.jp/update-dokosha
 Renban = 1
```

Comment:
Not something to change per se
but as discussed this will be changed into the future to use renban name+address instead of index number
or I'll either allow support for both 

I'll also deprecate the ``Pair`` key in the future
Since it can just be implicitly set in the code if the config file contains a ``Renban`` key. i.e.,

 ```py
pair = "Renban" in ballot_info
```